src/aap_eda/services/activation/engine/common.py

```diff
@@ -102,8 +110,8 @@ class ContainerRequest(BaseModel):
     name: str  # f"eda-{activation_instance.id}-{uuid.uuid4()}"
     image_url: str  # quay.io/ansible/ansible-rulebook:main
     cmdline: AnsibleRulebookCmdLine
-    activation_instance_id: str
-    activation_id: str
+    rulebook_process_id: str
+    resource_id: str
```

Comment:
`resource_id` is too generic, it has no clue the true meaning. In model RulebookProcess we call it parent or parent_type. In model ActivationRequestQueue we call it process_parent_id. To be consistent we can call it `process_parent_id` too here, because it refers to the same thing.

Reply:
Both id types are `int`, not `str`.

Reply:
Nice catch.  It was wrong even before me. 😉